pkg/controller/cassandracluster/reconcile_test.go

```diff
@@ -177,20 +177,15 @@ func TestFlipCassandraClusterUpdateSeedListStatusScaleDC2ManualSeedList(t *testi
 	assert.Equal(api.ClusterPhaseInitial, cc.Status.CassandraRackStatus["dc2-rack1"].CassandraLastAction.Name)
 	assert.Equal(3, len(cc.Status.CassandraRackStatus))
 
-	//cc.Status.SeedList = cc.InitSeedList()
-
-	var a = []string{
+	var firstSeedList = []string{
 		"cassandra-demo-dc1-rack1-0.nodomain.com",
 		"cassandra-demo-dc1-rack2-0.nodomain.com",
 		"cassandra-demo-dc2-rack1-0.nodomain.com",
 	}
+	cc.Status.SeedList = firstSeedList
 
 	assert.Equal(3, len(cc.Status.SeedList))
```

Comment:
@fdehay, not sure this test makes sense here. How could this list change after we've set it ?

Reply:
The copy of the initial status is made just before (line 189), so I thought we could simulate an update by changing it just after this copy. We then have a difference between the copier status and the cc.status.seedlist
Thanks for your time here, that's what I was looking for, I am not sure if what I did made sense